### PR TITLE
#3698 - Neverbounce new action

### LINKED ISSUE #3608: Docupilot
### App
Docupilot

### Actions
* Create Document

### LINKED ISSUE #3698: NeverBounce
### App
NeverBounce

### Actions
* Verify Email Address

### PR DESCRIPTION
**Pull request by GTFalcao**
# New Action
- **Verify Email Address**

Closes #3698 

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4414"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/neverbounce/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "dist"
+  ],
+  "homepage": "https://pipedream.com/apps/neverbounce",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@pipedream/platform": "^1.1.0",
```
Comment (feyzullah) [inline components/neverbounce/package.json:20]:
```suggestion
    "@pipedream/platform": "^1.1.1",
```
Let's have the latest version

File: components/neverbounce/app/neverbounce.app.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { defineApp } from "@pipedream/types";
+import { axios } from "@pipedream/platform";
+import {
+  HttpRequestParams,
+  VerifyEmailParams, VerifyEmailResponse,
```
Comment (feyzullah) [inline components/neverbounce/app/neverbounce.app.ts:5]: Can we put them all to the same line or can we have them all in different lines?

**Review by feyzullah — COMMENTED** [review]
Hi @GTFalcao this looks good to me. I added just two comments for minor optionals.

File: components/neverbounce/package.json (hunk 1 above)
Comment (GTFalcao) [reply components/neverbounce/package.json]: Updated the version! Thanks

File: components/neverbounce/app/neverbounce.app.ts (hunk 2 above)
Comment (GTFalcao) [reply components/neverbounce/app/neverbounce.app.ts]: I thought this was standardized with eslint but apparently it does not change this line.

I put them all on the same line. Thanks for noting!

**Review by feyzullah — APPROVED** [review]

**Pull request merged**